strings.txt

```diff
@@ -17810,3 +17810,601 @@
     sk = od %d
     es = desde %d
     pl = od %d
+
+  [meter]
+    en = m
+    ru = м
+    fr = m
+    da = m
+    id = m
+    ko = m
+    sv = m
+    tr = m
+    uk = м
+    vi = m
+    hu = m
+    de = m
+    fi = m
+    cs = m
+    it = m
+    nb = m
+    zh-Hans = m
+    th = ม.
+    ja = m
+    ro = m
+    ar = متر
+    sk = m
+    es = m
+    pl = m
+    nl = m
+
+  [kilometer]
+    en = km
+    ru = км
+    fr = km
+    da = km
+    id = km
+    ko = km
+    sv = km
+    tr = km
+    uk = км
+    vi = km
+    hu = km
+    de = km
+    fi = km
+    cs = km
+    it = km
+    nb = km
+    zh-Hans = km
+    th = กม.
+    ja = km
+    ro = km
+    ar = كم
+    sk = km
+    es = km
+    pl = km
+    nl = km
+
+  [placepage_place_description]
+    en = Description
+    ru = Описание
+    fr = Description
+    da = Beskrivelse
+    id = Deskripsi
+    ko = 설명….
+    sv = Beskrivning
+    tr = Açıklama
+    uk = Опис
+    vi = Mô tả
+    hu = Leírás
+    de = Beschreibung
+    fi = Kuvaus
+    cs = Popis
+    it = Descrizione
+    nb = Beskrivelse
+    zh-Hans = 说明
+    th = คำอธิบาย
+    ja = 説明
+    ro = Descriere
+    ar = الوصف
+    sk = Popis
+    es = Descripción
+    pl = Opis
+    nl = Beschrijving
+
+  [placepage_more_button]
```

Comment:
Там перевод с троеточиями, не факт что здесь они нужны.
> On Apr 26, 2016, at 4:48 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61087857>:
> 
> > +    de = Beschreibung
> > +    fi = Kuvaus
> > +    cs = Popis
> > +    it = Descrizione
> > +    nb = Beskrivelse
> > +    zh-Hans = 说明
> > +    th = คำอธิบาย
> > +    ja = 説明
> > +    ro = Descriere
> > +    ar = الوصف
> > +    sk = Popis
> > +    es = Descripción
> > +    pl = Opis
> > +    nl = Beschrijving
> > +
> > +  [placepage_more_button]
> Есть более точный перевод [bottom_sheet_more]
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/454ca48abdc1a7a643a178fac869c249f100121f..9460d6ca1e36b2c17bee62d5d712319e41eb236f#r61087857>